Audacia.Quest/Audacia.Quest.Core/Components/Transform.cs

```diff
@@ -6,7 +6,7 @@ public class Transform
     {
         public Vector2 Position { get; set; } = Vector2.Zero;
         public Vector2 Scale { get; set; } = Vector2.One;
-        public Vector2 Rotation { get; set; } = Vector2.Zero;
+        public float Rotation { get; set; } = 0;
```

Comment:
May want some xml documentation to state whether the rotation is in degrees or radians depending on how complex this is going to get.